codespell_lib/data/dictionary.txt

```diff
@@ -327,6 +328,8 @@ adapdive->adaptive
 adaped->adapted, adapt, adopted, adopt,
 adapive->adaptive
 adaptee->adapted
+adaption->adaptation
```

Comment:
These are both valid:
https://en.oxforddictionaries.com/definition/adaption
https://grammarist.com/usage/adaption-adaptation/

Reply:
I remove it.